deployment-aws/terraform/vars.tf

```diff
@@ -59,6 +59,32 @@ variable "git_commit" {
   default     = ""
 }
 
+variable "globus_endpoint_client_uuid" {
+  description = "The UUID for the Globus endpoint client"
+  type        = string
+}
+
+variable "globus_endpoint_client_secret" {
+  description = "The corresponding secret for the Globus endpoint client"
+  type        = string
+}
+
+variable "globus_app_client_uuid" {
+  description = "The UUID for the Globus application client"
+  type        = string
+}
+
+variable "globus_app_client_secret" {
+  description = "The corresponding secret for the Globus application"
+  type        = string
+}
+
+variable "globus_endpoint_id" {
+  description = "The endpoint UUID for the Globus shared collection."
+  type        = string
+}
+
```

Comment:
These should be alphabetized. Also, need to provide default values (`null`?) to make these variables optional.